front/src/modules/ui/input/relation-picker/components/SingleEntitySelect.tsx

```diff
@@ -13,15 +13,10 @@ import { isDefined } from '~/utils/isDefined';
 import { useEntitySelectSearch } from '../hooks/useEntitySelectSearch';
 import { EntityForSelect } from '../types/EntityForSelect';
 
-import { SingleEntitySelectBase } from './SingleEntitySelectBase';
-
-export type EntitiesForSingleEntitySelect<
```

Comment:
The same type exists in SingleEntitySelectBase so now we are simply importing it. I think it's acceptable since SingleEntitySelectBase is the "base" of SingleEntitySelect